### PR TITLE
Fix issues.

### PR DESCRIPTION
**Pull request by LiliDeng**

### CODE REVIEW

File: Libraries/CommonFunctions.psm1 [hunk 1]
```diff
@@ -2476,8 +2477,10 @@ Function Wait-AzVMBackRunningWithTimeOut($AllVMData, [scriptblock]$AzVMScript) {
         foreach ($vmData in $AllVMData) {
             $vmData.PublicIP = ($VMDataWithPublicIP | Where-Object {$_.RoleName -eq $vmData.RoleName}).PublicIP
         }
-
-        if ((Is-VmAlive -AllVMDataObject $AllVMData -MaxRetryCount 10) -eq "True") {
+        # the core is more, the boot time is longer
+        $MaxRetryCount = [int]($MaximumCores / 10) + 10
```
Comment (johnsongeorge-w) [inline Libraries/CommonFunctions.psm1:2481]: How did we come to this equation. Is it sufficient when the number of cores increases?

File: Testscripts/Windows/DiskHotAddRemove-Parallel.ps1 [hunk 2]
```diff
@@ -60,17 +60,27 @@ function Main {
             throw "Fail to attach $count empty data disks of size $diskSizeinGB GB to VM"
         }
         Write-LogInfo "Verifying if data disks are added to the VM - running fdisk on remote VM"
-        $fdiskOutput = Run-LinuxCmd -username $user -password $password -ip $AllVMData.PublicIP -port $AllVMData.SSHPort -command "/sbin/fdisk -l | grep /dev/sd" -runAsSudo
-        foreach ($line in ($fdiskOutput.Split([Environment]::NewLine))) {
-            if ($line -imatch "Disk /dev/sd[a-z][a-z]|sd[c-z]:" -and [int64]($line.Split()[4]) -eq (([int64]($diskSizeinGB) * [int64]1073741824))){
-                $verifiedDiskCount += 1
-                Write-LogInfo "$verifiedDiskCount data disk is successfully attached the VM: $line"
+        $retry = 1
+        $retryMaxTimes = 30
```
Comment (johnsongeorge-w) [inline Testscripts/Windows/DiskHotAddRemove-Parallel.ps1]: Please add a comment as why this retry is added.
Is there any particular observation to select 30*30

**Review by johnsongeorge-w — DISMISSED** [review]

File: Libraries/CommonFunctions.psm1 (hunk 1 above)
Comment (LiliDeng) [reply Libraries/CommonFunctions.psm1:2481]: we already tested 416 core, so far, it is the most core on azure?

File: Testscripts/Windows/DiskHotAddRemove-Parallel.ps1 (hunk 2 above)
Comment (LiliDeng) [reply Testscripts/Windows/DiskHotAddRemove-Parallel.ps1]: actually, retry once, it will break.

**Review by johnsongeorge-w — APPROVED** [review]

**Pull request merged**